chia/simulator/full_node_simulator.py

```diff
@@ -44,7 +91,7 @@ async def get_all_full_blocks(self) -> List[FullBlock]:
         return blocks
 
     @api_request
-    async def farm_new_transaction_block(self, request: FarmNewBlockProtocol):
+    async def farm_new_transaction_block(self, request: FarmNewBlockProtocol) -> FullBlock:
```

Comment:
These api requests are not designed to be called manually, they are meant to be called through the chia protocol (through server.py). So it's kind of weird that we are returning stuff from them.

When an API method does return something, it's usually a Message which then gets sent back over the wire. We are breaking that patter here.